packages/office-ui-fabric-react/src/components/Button/MessageBarButton/MessageBarButton.styles.ts

```diff
@@ -1,28 +1,25 @@
 import { IButtonStyles } from '../Button.types';
 import { ITheme, concatStyleSets, getFocusStyle } from '../../../Styling';
 import { memoizeFunction } from '../../../Utilities';
-import { getStyles as getBaseButtonStyles } from '../BaseButton.styles';
 
 export const getStyles = memoizeFunction(
-  (theme: ITheme, customStyles?: IButtonStyles, focusInset?: string, focusColor?: string): IButtonStyles => {
-    const baseButtonStyles: IButtonStyles = getBaseButtonStyles(theme);
-    const messageBarButtonStyles: IButtonStyles = {
-      root: [
-        getFocusStyle(theme, {
-          inset: 1,
-          highContrastStyle: {
-            outlineOffset: '-4px',
-            outlineColor: 'ActiveBorder',
+  (theme: ITheme, customStyles?: IButtonStyles): IButtonStyles =>
+    concatStyleSets(
+      {
+        root: [
+          getFocusStyle(theme, {
+            inset: 1,
+            highContrastStyle: {
+              outlineOffset: '-4px',
+              outlineColor: 'ActiveBorder',
+            },
+            borderColor: 'transparent',
+          }),
+          {
+            height: 24,
           },
-          borderColor: 'transparent',
-        }),
-        {
-          height: 24,
-          borderColor: theme.palette.neutralTertiaryAlt,
-        },
-      ],
-    };
-
-    return concatStyleSets(baseButtonStyles, messageBarButtonStyles, customStyles)!;
```

Comment:
Unless I'm missing something, there is no benefit to concatting `baseButtonStyles`, since this will also be done within the BaseButton component.

Reply:
It looks like all the button variants are doing a manual concat with the baseButtonStyles, but I'm not sure why... @dzearing do you know the history here?

Reply:
lgtm

Reply:
I don't fully remember the history.